posthog/tasks/usage_report.py

```diff
@@ -972,12 +970,12 @@ def _get_team_report(all_data: dict[str, Any], team: Team) -> UsageReportCounter
         dashboard_tagged_count=all_data["teams_with_dashboard_tagged_count"].get(team.id, 0),
         ff_count=all_data["teams_with_ff_count"].get(team.id, 0),
         ff_active_count=all_data["teams_with_ff_active_count"].get(team.id, 0),
-        hogql_app_bytes_read=all_data["teams_with_hogql_app_bytes_read"].get(team.id, 0),
-        hogql_app_rows_read=all_data["teams_with_hogql_app_rows_read"].get(team.id, 0),
-        hogql_app_duration_ms=all_data["teams_with_hogql_app_duration_ms"].get(team.id, 0),
-        hogql_api_bytes_read=all_data["teams_with_hogql_api_bytes_read"].get(team.id, 0),
-        hogql_api_rows_read=all_data["teams_with_hogql_api_rows_read"].get(team.id, 0),
-        hogql_api_duration_ms=all_data["teams_with_hogql_api_duration_ms"].get(team.id, 0),
+        query_app_bytes_read=all_data["teams_with_query_app_bytes_read"].get(team.id, 0),
+        query_app_rows_read=all_data["teams_with_query_app_rows_read"].get(team.id, 0),
+        query_app_duration_ms=all_data["teams_with_query_app_duration_ms"].get(team.id, 0),
+        query_api_bytes_read=all_data["teams_with_query_api_bytes_read"].get(team.id, 0),
```

Comment:
@patricio-posthog this will likely be your usage key for the API product - `query_api_bytes_read`